nano/core_test/node.cpp

```diff
@@ -368,7 +368,14 @@ TEST (node, receive_gap)
 				 .build_shared ();
 	node1.work_generate_blocking (*block);
 	nano::publish message{ nano::dev::network_params.network, block };
-	node1.network.inbound (message, node1.network.udp_channels.create (node1.network.endpoint ()));
+
+	// Another node instance just to be a tcp client for channel1.
+	auto other_node (std::make_shared<nano::node> (system.io_ctx, nano::get_available_port (), nano::unique_path (), system.logging, system.work));
+	other_node->start ();
+	system.nodes.push_back (other_node);
```

Comment:
Just a thought for the future and not part of this pull request: It is crazy that we have to create a full `node` if all we need is just a `channel_tcp`. `channel_tcp` should be constructable without requiring a `node` or `stat` instance.

Reply:
Agreed!